src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  padding: 0;
+  font-family: Avenir, serif;
+}
```

Comment:
Try to avoid selector *. Yes in these tasks it doesn't influence performance, but in bigger projects, it can load browser